src/kirby/components/modal/action-sheet/action-sheet.component.scss

```
@@ -18,4 +19,5 @@ kirby-card {
   margin-bottom: size("m");
   width: 160px; //TODO: Get width from common settings
   pointer-events: auto;
+  box-shadow: #{get-elevation('z8')};
```

No need to wrap the function call (only needed when assigning to css variables):
```suggestion
  box-shadow: get-elevation('z8');
```